models/category/src/category-draft/presets/sample-data-goodstore/bedding.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LocalizedString,
+  KeyReference,
+} from '@commercetools-test-data/commons';
+import type { TCategoryDraftBuilder, TCategoryDraft } from '../../../types';
+import empty from '../empty';
+import homeDecor from './home-decor';
+
+const homeDecorDraft = homeDecor().build<TCategoryDraft>();
+
+const bedding = (): TCategoryDraftBuilder =>
+  empty()
+    .name(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('Bedding')
+        ['en-US']('Bedding')
+        ['de-DE']('Bettwäsche')
+    )
+    .slug(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('bedding')
+        ['en-US']('bedding')
+        ['de-DE']('bettwasche')
+    )
+    .key('bedding')
+    .orderHint('.5')
+    .parent(KeyReference.presets.category().key(homeDecorDraft.key!));
```

Comment:
The parent in the data looks to be `furniture.ts`, but this makes more sense...am I missing something?

Reply:
Never mind, I was searching the CSV improperly